info.colobot.Colobot.yaml

```diff
@@ -0,0 +1,179 @@
+---
+app-id: info.colobot.Colobot
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: colobot
+rename-icon: colobot
+rename-desktop-file: colobot.desktop
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--device=dri"
+  - "--socket=pulseaudio"
+  - "--env=LD_LIBRARY_PATH=/app/lib64:/app/lib"
```

Comment:
Remove this and don't install anything in `lib64`.

Reply:
As noted on the other line about `lib64`, this is due to a bug in the libsndfile CMake lists, so I'm going to wait until the lack of ogg/vorbis is fixed in the runtime and then remove all mentions of it.